### DISCUSSION TITLE
--strip-unknown-mods lead to zero quantity

### DISCUSSION THREAD

**Discussion by kcasctivcr** [vdemichev/DiaNN#461] [Q&A] [answered]
Hi Vadim,

I run DIA-NN twice on a same DIA input file, in both runs, I create a spectral library with DIA-NN based on a fasta file, and the only difference between the two runs is whether I specified --strip-unkown-mods.
I noticed that some peptides can only be quantified if I specifies --strip-unkown-mods, but I cannot think of a good reason why.
I wonder what exactly does the command --strip-unkown-mods does?
And if a peptide in the output has 0 quantity, does this means that this peptide is detected but not able to quantify?

Best,
Hung

**Comment by vdemichev** [comment, marked as answer]
Hi Hung,

How does the log look like with and without this command?

Best,
Vadim

**Reply by kcasctivcr** [reply]
Hi Vadim,

The log is attatched below. Thanks!

[output.log.txt](https://github.com/vdemichev/DiaNN/files/9275322/output.log.txt)
[library.log.txt](https://github.com/vdemichev/DiaNN/files/9275323/library.log.txt)
[output_without_strip_unknown_mods.log.txt](https://github.com/vdemichev/DiaNN/files/9275324/output_without_strip_unknown_mods.log.txt)
[library_without_strip_unknown_mods.log.txt](https://github.com/vdemichev/DiaNN/files/9275326/library_without_strip_unknown_mods.log.txt)

Best
Hung

**Reply by vdemichev** [reply]
No fragments were generated for peptides with Sulfo and P(Oxidation) without --strip-unknown-mods. That means those peptides werre effectively skipped during search. Thus different output.

**Reply by kcasctivcr** [reply]
Thanks a ton!
